etn-ccis/blui-react-workflows
# Enhance example2 and restructure new architecture files

#### Changes proposed in this Pull Request:

- Enhance example2
- Restructure new architecture files



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- `yarn start:example2`
- navigate about and ensure everything that is demo-able is working properly



#### Any specific feedback you are looking for?

- I want to make sure this example project is building, that you understand how it is setup so you can make use of it for the refactor.
- Also look at the new folder structure and ensure you understand it and it is useful to everyone. We will need to move all of this again at the end, but for now we should be able to work out of new-architecture and example2 just fine.
